osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -256,7 +256,7 @@ public override void OnKilled()
 
         protected override void CheckForResult(bool userTriggered, double timeOffset)
         {
-            if (userTriggered || Time.Current < HitObject.EndTime)
+            if (userTriggered || !TailCircle.Judged)
```

Comment:
```suggestion
            if (userTriggered || Time.Current < HitObject.EndTime || !TailCircle.Judged)
```

Should fix hitsounds